e2e-tests/insurance/cypress/e2e/journeys/your-buyer/check-your-answers/check-your-answers-summary-list/check-your-answers-summary-list-total-contract-value-above-threshold.spec.js

```diff
@@ -1,18 +1,21 @@
-import { INSURANCE_ROUTES } from '../../../../../../../constants/routes/insurance';
-import { YOUR_BUYER as YOUR_BUYER_FIELD_IDS } from '../../../../../../../constants/field-ids/insurance/your-buyer';
+import { FIELD_IDS, ROUTES } from '../../../../../../../constants';
 import checkSummaryList from '../../../../../../../commands/insurance/check-your-buyer-summary-list';
 
 const {
   ROOT,
   YOUR_BUYER: {
     CHECK_YOUR_ANSWERS,
   },
-} = INSURANCE_ROUTES;
+} = ROUTES.INSURANCE;
 
 const {
-  PREVIOUS_CREDIT_INSURANCE_COVER_WITH_BUYER,
-  HAS_PREVIOUS_CREDIT_INSURANCE_COVER_WITH_BUYER,
-} = YOUR_BUYER_FIELD_IDS;
+  INSURANCE: {
+    YOUR_BUYER: {
+      PREVIOUS_CREDIT_INSURANCE_COVER_WITH_BUYER,
+      HAS_PREVIOUS_CREDIT_INSURANCE_COVER_WITH_BUYER,
+    },
+  },
+} = FIELD_IDS;
```

Comment:
`INSURANCE_ROUTES`, `INSURANCE_FIELD_IDS`, / field IDs from buyer - can be done in the next PR